src/components/authentication/Login.jsx

```diff
@@ -29,6 +33,10 @@ export const Login = () => {
           type: LOG_IN_SUCCESS,
           payload: { userToken: encodedToken, userData: foundUser },
         });
+        wishListDispatch({
+          type: INITIALIZE_WISH_COUNT,
+          totalProductsInwishList: foundUser.wishlist.length,
```

Comment:
This is also fine, but you should send your wishlist length in playload as it is a good practice and everyone is familiar with payload term. If you want to send multiple times then you can send your payload as an object too.